streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html

```diff
@@ -0,0 +1,20 @@
+<div id="topbar">
+    <div class="navbar">
+        <div class="navbar right"
+             align="right"></div>
```

Comment:
You probably don't need to/shouldn't break this into two lines - html doesn't follow Python style rules and newlines inside a tag is not great, especially for such short tags.